src/isEmpty.js

```diff
@@ -12,10 +12,13 @@ var _curry1 = require('./internal/_curry1');
  * @return {Boolean}
  * @example
  *
- *      R.isEmpty([1, 2, 3]); //=> false
- *      R.isEmpty([]); //=> true
- *      R.isEmpty(''); //=> true
- *      R.isEmpty(null); //=> false
+ *      R.isEmpty([1, 2, 3]);   //=> false
+ *      R.isEmpty([]);          //=> true
+ *      R.isEmpty('');          //=> true
+ *      R.isEmpty(null);        //=> false
+ *      R.isEmpty(R.keys({}));  //=> true
+ *      R.isEmpty({});          //=> false ({} does not have a length property)
```

Comment:
"`{}` is not list, i.e. not array-like" might be better here. i think specifying that it needs a `length` property is an implementation detail we should keep hidden.